From 24f28a78fdcd056357f137650ca7f0f01c257d97 Mon Sep 17 00:00:00 2001
From: Connor Behan <connor.behan@gmail.com>
Date: Wed, 19 Dec 2012 18:41:14 +0000
Subject: Remove call to obsolete miInitializeBackingStore()

Definition was deleted from Xorg during 1.14 merge window, but has been
a no-op since 1.10 merge window.

Signed-off-by: Connor Behan <connor.behan@gmail.com>
---
diff --git a/src/r128_driver.c b/src/r128_driver.c
index a062a60..f47c7e3 100644
--- a/src/r128_driver.c
+++ b/src/r128_driver.c
@@ -2701,7 +2701,6 @@ Bool R128ScreenInit(SCREEN_INIT_ARGS_DECL)
     R128DGAInit(pScreen);
 
 				/* Backing store setup */
-    miInitializeBackingStore(pScreen);
     xf86SetBackingStore(pScreen);
 
 				/* Set Silken Mouse */
--
cgit v0.9.0.2-2-gbebe
